### DISCUSSION TITLE
Private access roads

### DISCUSSION THREAD

**Discussion by scottclarke61** [valhalla/valhalla#3602] [Q&A]
I'm trying to work out routes for our emergency response cars that can use motorway access roads and gates marked as private. Even with "private_access_penalty":0, "gate_penalty":0,"service_cost":0 in the JSON load, the route only include roads marked as private access if it starts or finishes on that road. 

Is there a way to include all private or specific private roads for all routes?

**Comment by dnesbitt61** [comment]
Ultimately the best way I think would be to add an emergency vehicle costing class to support this. It could include all private roads as well as those that only allow emergency vehicle access.

**Comment by hup333** [comment]
Would it be an option to ignore all acces?
Emergency, like ambulance, fire truck, police, need to get to the destination to deal with the emergency
